docs/metadata-store/clustering.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Clustering and Khepri
+---
+
+import diagramStyles from './diagram.module.css';
+import MetadataStoreCluster from './metadata-store-cluster.svg';
+import MetadataStoreMinority from './metadata-store-minority.svg';
+
+# Clustering and Khepri
+
+When you cluster RabbitMQ, what really happens is that RabbitMQ calls the
+metadata store backend to create or expand its cluster. This is the same with
+both Mnesia and Khepri.
+
+Therefore, creating or expanding a Khepri-based RabbitMQ cluster is done the
+same way as with Mnesia.
+
+<figure className={diagramStyles.diagram}>
+<MetadataStoreCluster/>
+<figcaption>Clustering RabbitMQ happens at the metadata store level</figcaption>
+</figure>
+
+## Creating a cluster
+
+You can create a cluster using the regular methods:
+* using the CLI
+* using peer discovery
+
+See the [clustering guide](../clustering) for a complete description.
+
+Here is an example using the CLI:
+
+```
+rabbitmqctl join_cluster rabbit@remote-host
+```
+
+## Removing a node from a cluster
+
+Again, the regular methods can be used to remove a node from a cluster. Here
+is an example with the CLI:
+
+```
+rabbitmqctl stop_app
+rabbitmqctl reset
+rabbitmqctl start_app
+```
+
+## When to enable Khepri?
+
+It is possible to enable Khepri before or after you create or expand the
+RabbitMQ. When Khepri is enabled, the first step is to [synchronize the
+cluster view from Mnesia to
+Khepri](./enabling-khepri#cluster-membership-synchronization) if Khepri was
+enabled before the cluster was created.
+
+Also, if you want to cluster nodes where some of them are using Mnesia and
+some Khepri, the effective backend in the end will depend on several
+conditions, like if some nodes are already clustered. You will have to verify
```

Comment:
This paragraph about clustering a mix of khepri-enabled and not enabled nodes is confusing to me. Do we need to discuss this at all? If so, should we move this to a dedicated paragraph and elaborate when would someone need/want to do this?

Reply:
Ok, I deleted it for now.